src/compas_rhino/geometry/trimesh/slice.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import Rhino
+from Rhino.Geometry import Point3d, Vector3d, Plane
+from Rhino.Geometry.Intersect.Intersection import MeshPlane
+
+import compas
+from compas.plugins import plugin
+
+
+__all__ = [
+    'trimesh_slice',
+]
+
+
+@plugin(category="trimesh", requires=['Rhino'])
+def trimesh_slice(mesh, planes):
+    """Slice a mesh by a list of planes.
```

Comment:
I think there needs to be a blank line added for the docs to be rendered correctly.
```suggestion
    """Slice a mesh by a list of planes.
    
```